
float32 x_position
float32 y_position
float32 z_position
bool gripper_closed
bool wp_sensed
float32 x_max
float32 y_max
float32 z_max
bool final
uint32 msgid
uint32 status
